src/containers/Sitemap/Sitemap.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import { Contrast } from '@entur/layout'
+import {
+    Heading1,
+    Heading2,
+    Heading3,
+    Link,
+    Paragraph,
+} from '@entur/typography'
+import { Navbar } from '../Navbar/Navbar'
+import classes from './Sitemap.module.scss'
+
+const Sitemap: React.FC = () => (
+    <Contrast>
+        <Helmet>
+            <title>Mine tavler - Tavla - Entur</title>
+        </Helmet>
+        <Navbar />
+        <div className={classes.Wrapper}>
+            <Heading1>Nettstedkart</Heading1>
+            <Heading3>
+                I listen under kan du se hvordan nettsiden er bygd opp og hvor
+                sidene ligger i forhold til hverandre.
+            </Heading3>
```

Comment:
Litt usikker på heading her, informasjonen føles ikke helt ut som en "overskrift" for meg

Reply:
👍